src/sql/parts/dashboard/widgets/tasks/tasksWidget.component.ts

```diff
@@ -71,6 +71,8 @@ export class TasksWidget extends DashboardWidget implements IDashboardWidget, On
 		let tasksConfig = this._config.widget[selector] as Array<string | ITask>;
 		let tasks = TaskRegistry.getTasks();
 
+		// disable filter temporarily in order to show all tasks registered.
```

Comment:
Is this meant to be disabled? I don't think we want to disable a feature.

Reply:
👍

Reply:
1. Review with PM (vicky, alan)
2. Show screenshots so all can review
3. Add your tasks to the default list instead of disabling this feature entirely

Reply:
Worst case, we can have a separate tab for cluster contributed, put all our stuff there, and it should only show for big data cluster connection.

Long term we know we need to fix home page so it's adaptable but right now it's really not. Users can modify but it's hard for extension to.

Reply:
fixed.